app/components/spotlight/analytics/dashboard_component.html.erb

```diff
@@ -0,0 +1,51 @@
+<%= cache current_exhibit, expires_in: 1.hour do %>
```

Comment:
This cache key/block is problematic. On the first render of this partial for an exhibit it will cache the whole partial and then display the same content no matter what start_date or end_date is applied in later requests. I think to fix this you'd need to add the start/end param values to the cache key or remove the conditional display for now, which is what I suggest below.

Reply:
This is not the behavior I am seeing in the app. It isn't going to be relevant because I am going to remove the start date, but it does reload with the right variables when you add a start date. My assumption was this line makes it work https://github.com/projectblacklight/spotlight/blob/539d806a2d8b90e618ef18e54070b8ea99edb2a3/app/models/concerns/spotlight/exhibit_analytics.rb#L11.

Definitely could be a problem if there is a different end_date and no new start date. Anyway moot for now due to the removal of the start/end.